safe-global/web-core
# fix: adjust SDK error message + fix onboard

## What it solves

Resolves unclear error message on older Safes

## How this PR fixes it

When the Core SDK is not initialized (it only supports >= v1.1.1 Safes), the error message is now "The Safe SDK could not be initialized. Please be aware that we only support >= v1.1.1 Safes." when transacting.

Note: the temporary `@web3-onboard` fix for `joi` schemas has been installed as well. This should be upgraded once they next publish a release.

## How to test it

- Open a v1.0/1.1.0 Safe and create a transaction. Observe the error message.
- Open a >= v1.1.1 Safe then switch to a < v1.1.1 Safe and repeat the above. Observe the error message.
